odl/operator/default_ops.py

```diff
@@ -1087,18 +1123,23 @@ def adjoint(self):
 
 class ComplexEmbedding(Operator):
 
-    """Operator that embeds a vector into a complex space."""
+    """Operator that embeds a vector into a complex space.
+
+    Implements::
+
+        ComplexEmbedding(x) == x + 1j * zero()
+    """
 
     def __init__(self, space, scalar=1):
         """Initialize a new instance.
 
         Parameters
         ----------
-        space : `FnBase`
-            Space which real part should be taken, needs to implement
+        space : `TensorSpace`
+            Space whose into which should be embedded, needs to implement
             ``space.complex_space``.
         scalar : ``space.complex_space.field`` element, optional
-            Scalar which the incomming vectors should be multiplied by in order
+            Scalar to be multiplied with incoming vectors in order
```

Comment:
Why do we have this parameter? IMO it's not the responsibility of this operator to multiply with anything, that should be done separately.